Dockerfile

```diff
@@ -1,20 +1,13 @@
 FROM ubuntu:16.04
 
-LABEL maintainer="zyxep"
+RUN apt-get update
+RUN apt-get install -y wget curl
+RUN /bin/bash -c "$(wget -qO- https://apt.vapor.sh)"
 
-# Install CURL
-RUN apt-get update && \
-    apt-get -y install curl && \
-    rm -rf /var/lib/apt/lists/*;
-
-# Get Vapor repo including Swift
-RUN curl -sL https://apt.vapor.sh | bash;
-
-# Installing Swift & Vapor
-RUN apt-get update && \
```

Comment:
We dont need to update apt-get again as the vapor script does it and the script before to install curl/wget does it as well.